scripts/cmd/build.js

```diff
@@ -109,5 +109,5 @@ export default async function build(...args) {
 }
 
 async function clean(outdir) {
-	return del([`${outdir}/**`, `!${outdir}/**/*.d.ts`]);
+	return deleteAsync([`${outdir}/**`, `!${outdir}/**/*.d.ts`]);
```

Comment:
Not sure if this is relevant here: https://github.com/sindresorhus/del#beware

Doesn't appear to be a breaking change from prior version: https://github.com/sindresorhus/del/tree/v6.1.1#beware

... So it should be behaving the same as it was before this PR.